Bookish/Bookish.DataAccess/BookCopies.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Bookish.DataAccess
+{
+    public class BookCopies
```

Comment:
Small change: might want to rename this to BookCopy per the convention that a class has a singular name (whereas a DB table has plural). Easy to forget I know!